content_sync/apis/github.py

```diff
@@ -399,3 +399,26 @@ def commit_tree(self, element_list: [InputGitTreeElement], user: User) -> Commit
         )
         main_ref.edit(commit.sha)
         return commit
+
+    def get_all_file_paths(self, path: str):
```

Comment:
Can we add a return type too? `Iterable[str]` should work fine here according to the docs https://docs.python.org/3/library/typing.html#typing.Generator